libraries/AP_Soaring/AP_Soaring.cpp

```diff
@@ -472,4 +479,14 @@ bool SoaringController::check_drift(Vector2f prev_wp, Vector2f next_wp)
     }
 }
 
+float SoaringController::get_thermalling_radius() const
+{
+    // Thermalling radius is controlled by parameter SOAR_THML_BANK and true target airspeed.
+    const AP_AHRS &_ahrs = AP::ahrs();
+    float target_aspd = _spdHgt.get_target_airspeed() * _ahrs.get_EAS2TAS();
+
+    float radius = target_aspd*target_aspd / (GRAVITY_MSS * tanf(thermal_bank*DEG_TO_RAD));
```

Comment:
const float

Also, please make the squaring more obvious with (target_aspd*target_aspd)